src/skdh/completeness/parse.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+import pandas as pd
+import warnings
+import re
+import os
+from skdh.completeness.helpers import from_unix, convert_sfreq_to_sampling_interval
+
+def vivalink_parse_ecg_data(df, ecg_key, s_freq):
```

Comment:
These device specific parsers should be wrapped into the CSV reader somehow, or if not possible, we can discuss how to include them or rely on the end-user to generate the inputs for this

Reply:
These parsers are more like helper functions for me or other people working with these devices to help getting from native to the forced input format, they're not actually a part of the pipeline. The native format changes a lot depending on software versions, data capture platform etc. so they should not be integrated directly in the pipeline or they will fail every single time a new data batch comes. We could possibly move this to  a private extension repo if we don't want to have it here.

Reply:
Removed the file from the repo, I believe we have an internal private skdh extension repo already that we can add it to?